app/scripts/services/routes.js

```diff
@@ -72,6 +73,10 @@ angular.module("openshiftConsole")
           }
           warnings.push(message);
         }
+
+        if (wildcardPolicy === 'Subdomain' && ingress.wildcardPolicy !== wildcardPolicy) {
+          warnings.push('Router "' + ingress.routerName + '" does not support wildcard subdomains. Your route will only be available at host ' + ingress.host + '.');
```

Comment:
Thanks, pulled the latest router image, and I see it now. It looks like we handle the case for both older and newer router images OK.

![screen shot 2016-11-11 at 8 42 10 am](https://cloud.githubusercontent.com/assets/1167259/20216850/d0ceba08-a7ea-11e6-8b5f-d8c4e38ecfab.png)